cross-chain/arbitrum/contracts/interfaces/ITokenBridge.sol

```diff
@@ -0,0 +1,216 @@
+// contracts/Bridge.sol
```

Comment:
I propose to move all `I*.sol` files directly to the `contracts` directory. This way they will be close to the code that uses them. Then, we can remove the `interfaces` dir. We use that approach in the core `solidity` package. I think the `interfaces` directory is typically used to group interfaces exported by the package, supposed to be used by outside clients. This is not the case here.

Reply:
Sure, we can move it, but let's wait on a moving part once we have a generic Wormhole Gateway implemented. Maybe it turns out that we won't need them.

Reply:
I can confirm we do not need them. The interface needed for Wormhole integration is embedded in `L2WormholeGateway` implemented in #570.

Reply:
Good, all the interfaces are removed now [f958a53](https://github.com/keep-network/tbtc-v2/pull/554/commits/f958a538afa7ebe3544b895f4a7bfe89ce56e56c)